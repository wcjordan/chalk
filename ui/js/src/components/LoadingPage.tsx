import React from 'react';
import { StyleSheet, ViewStyle } from 'react-native';
import { ActivityIndicator } from 'react-native-paper';

interface Style {
  loadingIndicator: ViewStyle;
}

const styles = StyleSheet.create<Style>({
  loadingIndicator: {
    margin: 'auto',
  },
});

const LoadingPage: React.FC = function () {
  return (
    <React.Fragment>
      <ActivityIndicator size={'large'} style={styles.loadingIndicator} />
    </React.Fragment>
  );
};

export default LoadingPage;
