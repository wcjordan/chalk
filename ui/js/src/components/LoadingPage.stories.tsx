import React from 'react';
import LoadingPage from './LoadingPage';

export default {
  title: 'Loading Page',
  component: LoadingPage,
};
export const DefaultLoadingPage: React.FC = () => <LoadingPage />;
