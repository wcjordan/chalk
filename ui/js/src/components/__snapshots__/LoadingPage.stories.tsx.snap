// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Page DefaultLoadingPage smoke-test 1`] = `
<div class="css-g5y9jx r-13awgt0">
  <div class="css-g5y9jx r-13awgt0 r-12vffkv">
    <div role="progressbar"
         class="css-g5y9jx r-1awozwy r-1777fci r-13awgt0 r-ywje51"
    >
      <div class="css-g5y9jx"
           style="width: 48px; height: 48px; opacity: 1;"
      >
        <div class="css-g5y9jx r-1awozwy r-1p0dtai r-1777fci r-1d2f490 r-u8s1d r-zchlnj r-ipm5af">
          <div class="css-g5y9jx"
               style="width: 48px; height: 48px; transform: rotate(mocked_deg);"
          >
            <div class="css-g5y9jx"
                 style="width: 48px; height: 24px; overflow: hidden;"
            >
              <div class="css-g5y9jx"
                   style="width: 48px; height: 48px; transform: translateY(0px) rotate(mocked_deg);"
              >
                <div class="css-g5y9jx"
                     style="width: 48px; height: 24px; overflow: hidden;"
                >
                  <div class="css-g5y9jx"
                       style="width: 48px; height: 48px; border-color: rgb(103, 80, 164); border-width: 4.8px; border-radius: 24px;"
                  >
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class="css-g5y9jx r-1awozwy r-1p0dtai r-1777fci r-1d2f490 r-u8s1d r-zchlnj r-ipm5af">
          <div class="css-g5y9jx"
               style="width: 48px; height: 48px; transform: rotate(mocked_deg);"
          >
            <div class="css-g5y9jx"
                 style="width: 48px; height: 24px; overflow: hidden; top: 24px;"
            >
              <div class="css-g5y9jx"
                   style="width: 48px; height: 48px; transform: translateY(-24px) rotate(mocked_deg);"
              >
                <div class="css-g5y9jx"
                     style="width: 48px; height: 24px; overflow: hidden;"
                >
                  <div class="css-g5y9jx"
                       style="width: 48px; height: 48px; border-color: rgb(103, 80, 164); border-width: 4.8px; border-radius: 24px;"
                  >
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
